docs/components/connectors/out-of-the-box-connectors/uipath.md

```diff
@@ -0,0 +1,243 @@
+---
+id: uipath
+title: UIPath Connector
+description: Orchestrate your UIPath Bots with Camunda. You can create new queue items and get the result from it.
+---
+
+The **UIPath Connector** allows you to connect your BPMN process with [UIPath](https://cloud.uipath.com) orchestrator. 
```

Comment:
maybe orchestrate UIPath bot instead of connect?

Reply:
Done